src/config-ci.py

```diff
@@ -10,8 +10,10 @@
 NO_GIT = True
 
 # test_verify_schema.sh might have checked this out
-if os.path.isdir("/tmp/contact"):
+if os.path.isdir("/tmp/contact") and os.path.exists("/tmp/contact/contacts.yaml"):
```

Comment:
```suggestion
if os.path.exists("/tmp/contact/contacts.yaml"):
```

seems like the `os.path.isdir("/tmp/contact")` part is redundant now that you're checking for the file itself?